### PR TITLE
link to delegates at the dlang specification

### PR DESCRIPTION
**Pull request by wilzbach**

### CODE REVIEW

File: public/content/en/basics.md [hunk 1]
```diff
@@ -1177,15 +1177,19 @@ Nameless function which are called *lambdas* can be defined in two ways:
...
 
+### In-depth
+
+- [Delegate's specification](https://dlang.org/spec/function.html#closures)
+
 ## {SourceCode}
 import std.stdio;
 
-enum IntOps {
+enum IntOps = {
```
Comment (stonemaster) [inline public/content/en/basics.md]: That's the old non-working example

File: public/content/en/basics.md [hunk 2]
```diff
@@ -1220,13 +1224,13 @@ void main()
     int a = 10;
     int b = 5;
 
-    auto func = getMathOperation(IntOps.add);
+    auto func = getBusinessLogic(0);
```
Comment (stonemaster) [inline public/content/en/basics.md]: That's the old non-working example

File: public/content/en/basics.md [hunk 3]
```diff
@@ -1220,13 +1224,13 @@ void main()
...
     writeln("The type of func is ",
         typeof(func).stringof, "!");
 
     // run the delegate func which does all the
     // real work for us!
-    writeln("result: ", func(a, b));
+    writeln(func(a, b));
```
Comment (stonemaster) [inline public/content/en/basics.md]: ditto

**Comment by wilzbach** [comment]
rebased

**Pull request merged**